CMakeLists.txt

```diff
@@ -65,7 +71,10 @@ add_custom_target(all_xml_lint)
 add_subdirectory(utils)
 add_subdirectory(vpr)
 add_subdirectory(yosys)
+if (NOT DEFINED ENV{CI} OR NOT $ENV{CI})
```

Comment:
Can you add a comment here saying why this is disabled on CI?

Reply:
Done